thunder/core/transforms.py

```diff
@@ -1337,7 +1337,7 @@ def _grad_out_specifier_default(pytree: Any) -> list[TensorProxy]:
 # The algorithm for modifying the program has the following steps:
 #   1) Flattens the original trace for the grad transform -- ensuing that all top-level symbols have
 #       a grad function.
-def grad(
+def __grad(
```

Comment:
"register_grad" rules are still being used but now though the same code path as the rest of the thunder.jit with disable_torch_autograd=False.